modules/auth0/tenant/README.md

```diff
@@ -0,0 +1,165 @@
+# Component: `auth0/tenant`
```

Comment:
See README suggestions from `app` component

Reply:
and reference links